utilities/src/main/java/io/syndesis/qe/hooks/IssueHooks.java

```diff
@@ -37,7 +40,46 @@
 @Slf4j
 public class IssueHooks {
 
-    enum IssueState {
+    public static class SimpleIssue {
```

Comment:
you can use

```
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
```

Reply:
Cool, didn't know about that, will change. Also, maybe I should refactor this outside of `IssueHooks`, wdyt?

Reply:
or theoretically 

```
@Data
@AllArgsConstructor
```

may be enough - now I'm wondering what the [@RequiredArgsConstructor](https://projectlombok.org/features/constructor) will do when there are no final/@NonNull fields - will it create `@NoArgsConstructor`? :thinking: 

anyway yes, a separate class would be nice